LICENSE

```diff
@@ -279,6 +279,13 @@ SOURCE/JAVA-CORE
     This product contains lpad and rpad methods adapted from Apache Flink.
       * core/src/main/java/org/apache/druid/java/util/common/StringUtils.java
 
+    This product contains Scala logging and serializable Hadoop configuration utilities adapted from Apache Spark.
```

Comment:
I forgot to mention about license and notice. Thank you for adding them! This license and notice files are important for the release and must be included in our release artifacts. I'm not sure how we are going to release this spark extension yet, as it could be just a part of our regular release or it could be released separately just like [pydruid](https://github.com/druid-io/pydruid). So, this change LGTM for now, but we may need to move this license and notice change to somewhere else later depending on how we release this spark extension.